packages/strapi-plugin-graphql/services/Resolvers.js

```diff
@@ -68,11 +70,11 @@ module.exports = {
         Object.assign(acc.resolver[globalId], {
           createdAt: (obj) => {
             // eslint-disable-line no-unused-vars
-            return obj.createdAt || obj.created_at;
+            return isArrayTimestamps ? obj[_.get(model, 'options.timestamps[0]')] : obj.createdAt || obj.created_at;
           },
           updatedAt: (obj) => {
             // eslint-disable-line no-unused-vars
-            return obj.updatedAt || obj.updated_at;
+            return isArrayTimestamps ? obj[_.get(model, 'options.timestamps[1]')] : obj.updatedAt || obj.updated_at;
```

Comment:
Here we already check that `options.timestamps` is an array so we don't need to check it again.
So we just need to return `obj[_.get(model, 'options.timestamps[1]')]`